Integrations/integration-Netskope.yml

```diff
@@ -0,0 +1,307 @@
+        case 'test-module':
+            if (sendRequest('GET', "https://" + url + "/api/v1/events?token=" + token + "&type=application&timeperiod=3600&limit=1")) {
+                return 'ok';
+            }
+            return 'not cool';
+        case 'NetskopeEvents':
+            if (args.timeperiod == null && args.query == null) {
+                result = sendRequest('GET', "https://" + url + "/api/v1/events?token=" + token + "&type=" + encodeURIComponent(args.type) + "&starttime=" + encodeURIComponent(args.starttime) + "&endtime=" + encodeURIComponent(args.endtime));
+            } else if (args.timeperiod == null && args.query != null) {
+                result = sendRequest('GET', "https://" + url + "/api/v1/events?token=" + token + "&type=" + encodeURIComponent(args.type) + "&starttime=" + args.starttime + "&endtime=" + args.endtime + "&query=" + encodeURIComponent(args.query));
+            } else if (args.query == null) {
+                if (args.timeperiod == null){
+                    result = sendRequest('GET', "https://" + url + "/api/v1/events?token=" + token + "&type=" + encodeURIComponent(args.type) + "&starttime=" + arg.starttime + "&endtime=" + args.endtime);
+                } else {
+                result = sendRequest('GET', "https://" + url + "/api/v1/events?token=" + token + "&type=" + encodeURIComponent(args.type) + "&timeperiod=" + args.timeperiod);
+                }
+            } else {
+                result = sendRequest('GET', "https://" + url + "/api/v1/events?token=" + token + "&type=" + encodeURIComponent(args.type) + "&timeperiod=" + args.timeperiod + "&query=" + encodeURIComponent(args.query));
+            }
+
+            var keys = ["App","Timestamp","Activity","Object","Hostname","Category","DeviceClassification","User","FromUser","ToUser","ScrIP","AccessMethod","Url","AppCategory"];
+            var retArray = [];
+            result.obj.data.forEach( function (arrayItem)
+            {
+                var app =  arrayItem.app;
+                var timestamp =  arrayItem.timestamp;
+                var activity =  arrayItem.activity;
+                var object =  arrayItem.object;
+                var hostname =  arrayItem.hostname;
+                var category =  arrayItem.category;
+                var device_classification =  arrayItem.device_classification;
+                var user =  arrayItem.user;
+                var from_user =  arrayItem.from_user;
+                var to_user =  arrayItem.to_user;
+                var srcip =  arrayItem.srcip;
+                var access_method =  arrayItem.access_method;
+                var url =  arrayItem.url;
+                retArray.push({"App" : app, "Timestamp" : timestamp, "Activity" : activity, "Object" : object, "Hostname" : hostname, "Category" : category, "Device_Classification" : device_classification, "User" : user, "From_User" : from_user, "To_User" : to_user, "SourceIP" : srcip, "Access_Method" : access_method, "URL" : url})
```

Comment:
please format all the JSON/objects/maps not in one line, but each key, new line
example. Much easier to read and debug
NOT `{key1:val1, key2: val2}`
DO 
```
{
    key1: val1,
    key2: val2
}
```